src/components/application_manager/test/CMakeLists.txt

```diff
@@ -129,12 +141,29 @@ set(LIBRARIES
   jsoncpp
 )
 
-create_test(am_commands_test "${COMMANDS_SOURCES}" "${LIBRARIES}")
+add_library(mock_message_helper_library ${MOCK_MESSAGE_HELPER_SOURCE})
+target_link_libraries(mock_message_helper_library ${LIBRARIES})
```

Comment:
@AGritsevich I think this linkage is redundant. `LIBRARIES` will be linked to executable file directly.